interpreter/code_interpreters/dockerfiles/dev_note.md

```diff
@@ -0,0 +1,22 @@
+=======================================================================
+                              NOTE TO DEVELOPERS
+=======================================================================
+
+Please avoid manually editing the following files:
+- Dockerfile
+- requirements.txt
+- hash.json
+
+These files are key components for Open-Interpreter's containerized execution features. Manually editing them can disrupt the program's ability to:
+
+1. Know when to rebuild the Docker image.
+2. Perform other related functionalities efficiently.
+
+If you need to make adjustments, kindly use the 'DockerManager' class. It offers convenient methods like:
+- add_dependency
+- remove_dependency
+- add_language
```

Comment:
If `add_language` doesn't exist anymore we can remove the reference to it.

Reply:
I'm going to resolve this one and we can focus on it as a follow up once we get this MVP merged in and don't have to fight against merge conflicts with such a large changeset.